assets/xml/objects/object_an2.xml

```diff
@@ -1,12 +1,12 @@
 ﻿<Root>
     <File Name="object_an2" Segment="6">
         <DList Name="object_an2_DL_000370" Offset="0x370" />
-        <DList Name="object_an2_DL_000378" Offset="0x378" />
+        <DList Name="object_an2_DL_000378" Offset="0x378" /> <!-- Original name is "an_kasa_model" ("umbrella; parasol") -->
```

Comment:
interesting the animation is not prefixed with `an2`

Reply:
same for `an3` and `an4`

Reply:
I observed this with some objects in OoT too, mainly the various `ganon_anime`s. While they certainly use a "prefix + name" scheme for most assets, that prefix is definitely *not* always based on the object name, or even *unique* to that object! In this case, it looks like they used `anj` as a prefix for the animations that are used when Anju and Kafei finally meet up, and `an` for everything else. I don't know why they decided to make those specific animations have a separate prefix, but they did.

Reply:
Oh, and one important thing I forgot to mention; in MM3D, `an2`, `an3`, and `an4` simply don't exist. They are all combined into `an1`. Something similar happens when the `ganon_anime`s in OoT3D, though bizarrely they *did* create a brand-new object called `zelda_ganon_demo` *just* to handle the Ganondorf that appears before the Ganon fight. So I think in cases like Anju, Ganondorf, or Skull Kid, where the assets get spread across multiple objects, they conceptually treated them like one big object, and then split things out afterwards. Grezzo simply came in and combined them again.